components/camera/videosourcewrappers.go

```diff
@@ -198,16 +198,7 @@ func (vs *videoSource) Stream(ctx context.Context, errHandlers ...gostream.Error
 }
 
 func (vs *videoSource) Image(ctx context.Context, mimeType string, extra map[string]interface{}) ([]byte, ImageMetadata, error) {
-	img, release, err := ReadImage(ctx, vs.videoSource)
-	if err != nil {
-		return nil, ImageMetadata{}, err
-	}
-	defer release()
-	outBytes, err := rimage.EncodeImage(ctx, img, mimeType)
-	if err != nil {
-		return nil, ImageMetadata{}, err
-	}
-	return outBytes, ImageMetadata{MimeType: mimeType}, nil
+	return ReadImageBytes(ctx, vs.videoSource, mimeType)
```

Comment:
Could we move it into videosourcewrapper's as a public function instead of having it in camera.go? Then we can quarantine it with the rest of the gostream stuff until we can get rid of it?

Reply:
yep

Reply:
Done